aloe_webdriver/screenshot_failed.py

```diff
@@ -47,6 +58,12 @@
 
 FORMAT = 'failed_{feature_file}_{scenario_index}_{scenario_name}{outline_index}'
 
+# This constant should be redefined externally. Due to how `run_features` in
+# `aloe.testing.FeatureTest.run_features` reload modules on each run, this check
+#  prevents it to be reset.
```

Comment:
"prevents it from being reset" (but irrelevant, because the `globals` trick isn't needed)